homework/01LessonOne.js

```diff
@@ -106,7 +107,13 @@ var notNotTrue;
 // What is the difference between the "and" and the "or" operators? Explain why you would use each of them.
 //
 // Your Answer Goes Here:
-//
+// The && operator tests to see if all values are 'true', and if any values are 'false', the variable will return 'false'.
+// The || operator, however, tests to see if any values are 'true', and if at least one value is 'true', will return 'true'.
+// For the examples pertaining to this lesson, we are assuming that only two values are being compared.
+// NOTE:
+// When one uses multiple logical operators to test more than two values, operational precedence must be considered, as it makes things a little more complicated.
+// MDN example: the statement 'true && true || false' will return 'false', since || operators have higher operational precedence than && operators,
+// whereas the statement 'true && (true || false)' will return 'true', since (grouped) values have the absolute lowest operational precedence.
 // ****
```

Comment:
Good job. Not sure that using the word **variable** in that first sentence makes sense. But on the right track.